app/controllers/appeal-application/pay-now.ts

```diff
@@ -87,13 +106,60 @@ function postPayNow(updateAppealService: UpdateAppealService) {
       };
       let redirectPage = getRedirectPage(editingMode, editingModeRedirect, req.body.saveForLater, defaultRedirect);
 
-      return res.redirect(redirectPage);
+      if (['protection'].includes(appeal.application.appealType)) {
+        let isPcqUp: boolean = await checkPcqHealth();
+        if (isPcqUp) {
+          invokePcq(res, appeal);
+        } else {
+          return res.redirect(redirectPage);
+        }
+      } else {
+        return res.redirect(redirectPage);
+      }
     } catch (error) {
       next(error);
     }
+
   };
 }
 
+async function checkPcqHealth(): Promise<boolean> {
```

Comment:
I think we can move this one `checkPcqHealth` and next function `invokePcq` into a separate file ot be reused for the upcoming story RIA-5129.